docs/api.rst

```diff
@@ -844,6 +844,20 @@ The path is encoded if needed.
 
    .. versionadded:: 1.9
 
+.. method:: URL.__truediv__(url)
```

Comment:
@commonism since you documented this, could you also edit the mentions of `__truediv__` in https://github.com/aio-libs/yarl/blob/master/CHANGES.rst to make them linkable like in your changelog fragment.

Additionally, add the second change note named `1026.doc.rst` mentioning that this has been documented.